content/efcore/getting-started/samples/manipulating-entities/add-new-entity-to-existing/Program.cs

```diff
@@ -25,8 +25,10 @@ public static void Main()
             var addedBook = context.Books
                 .Include(b => b.Author)
                 .Single(b => b.Title.Contains("Badge"));
+			
+			Console.WriteLine("We added a book to the author {0} {1} -", author.FirstName, author.LastName);
 
```

Comment:
Remove the second `Console.WriteLine` that prints out the books info. It isn't necessary for this example.